v2/gcs-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/transforms/GcsToSourceStreamer.java

```diff
@@ -74,6 +93,7 @@ public void setup() {
     while (retry) {
       try {
         spannerDao = new SpannerDao(spannerConfig, tableSuffix, isMetadataDbPostgres);
+        storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
```

Comment:
Can this be done only when the writeFilteredEvents is true? Else it's an unnecessary step